microsoft-edge/devtools-guide-chromium/issues/index.md

```diff
@@ -96,21 +110,26 @@ Each displayed issue has the following components:
 
 An issue in the **Issues** tool may include one or more links that open different tools, such as the **Elements**, **Sources**, or **Network** tool. You can open one of these tools to perform additional troubleshooting steps.
 
-To open a linked tool from the **Issues** tool:
+To open a linked tool from an entry in the **Issues** tool:
 
-1. As described in the previous section, open the demo page and then expand an issue in the **Issues** tool.
+1. As described above, open a webpage, open DevTools, select the **Issues** tool, and then expand an issue.
 
-1. In **AFFECTED RESOURCES** > **Open in**, select the tool name.  The affected resource is displayed in the selected tool:
+<!-- Elements link not present
+1. In **AFFECTED RESOURCES** > **Open in**, click the tool name, such as **Elements**:
 
    ![Select a tool, to open an affected resource from within the Issues tool](./index-images/affected-resource-opens-elements-tool.png)
+-->
```

Comment:
```suggestion
   <!-- Elements link not present
   1. In **AFFECTED RESOURCES** > **Open in**, click the tool name, such as **Elements**:
   ![Select a tool, to open an affected resource from within the Issues tool](./index-images/affected-resource-opens-elements-tool.png)
   -->
```
existing tagging (comment at col 1, includes a blank line) causes reset to 1.  Will try indent + remove blank line, use VS Code's preview.  @captainbrosset